pkg/apis/testharness/v1beta1/command.go

```diff
@@ -0,0 +1,51 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import "strings"
+
+func (c *Command) String() string {
+	if c.Command == "" && c.Script == "" {
+		return "(invalid command with neither Command nor Script set)"
```

Comment:
This seems like an error message but is part of the string.  Are we trying to avoid err messages and error handling on the calling side?   Are we going to run into case of uses where the caller is going to need to scan for these error messages?

Reply:
I don't intend the value returned by `String` to be machine-readable. I'm just trying to make sure that if for some reason in the future an invalid `Command` object is included in the `kuttl` log, then its description will be useful to a human. I feel like such message does this, unlike a simple empty string.

FTR, neither this nor the next case should ever happen in real life, since we already reject `Command` objects where `.Command == "" XOR .Script == ""` isn't met during unmarshalling. But better safe than sorry.